backend/pastes/models.py

```diff
@@ -1,3 +1,35 @@
+"""Pastes models."""
+
+# Django
 from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+class PasteBin(models.Model):
```

Comment:
I would use another class name to avoid future misunderstandings - snippet, paste, fragment or something similar, maybe even something that will reflect that it will have a picture soon(no idea what would suit it)